vochain/indexer/process.go

```diff
@@ -144,21 +144,14 @@ func (idx *Indexer) newEmptyProcess(pid []byte) error {
 	}
 
 	eid := p.EntityId
-	// Get the block time from the Header
-	currentBlockTime := time.Unix(idx.App.TimestampStartBlock(), 0)
-
-	// If startBlock is zero, then we use the current block height (zero means start right now)
-	if p.StartBlock == 0 {
-		p.StartBlock = idx.App.Height()
-	}
 
 	// Create and store process in the indexer database
 	procParams := indexerdb.CreateProcessParams{
 		ID:                pid,
 		EntityID:          nonNullBytes(eid),
-		StartBlock:        int64(p.StartBlock),
-		EndBlock:          int64(p.BlockCount + p.StartBlock),
-		BlockCount:        int64(p.BlockCount),
+		StartDate:         time.Unix(int64(p.StartTime), 0),
```

Comment:
Why not using `TimestampToTime` function recently created ?
Same for all similar operations.